Sources/Core/Builtin.swift

```diff
@@ -0,0 +1,9 @@
+// Copyright © 2022 Max Desiatov <max@desiatov.com>.
+// All Rights Reserved.
+// SPDX-License-Identifier: BSD-3
+
+@inlinable
+@inline(__always)
+func minAllocationAlignment() -> Int {
```

Comment:
Is this supposed to be `public` or `internal`?

Reply:
It's `internal`, do you want that to be explicit?

Reply:
Yes please